net-libs/libp8_platform/libp8_platform-2.1.0.1.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="Platform support library used by libCEC and binary add-ons for Kodi"
+DESCRIPTION="This library provides platform specific support for other libraries"
+HOMEPAGE="http://libcec.pulse-eight.com/"
+COPYRIGHT="2011-2015 Pulse-Eight Limited"
+LICENSE="GNU GPL v2"
+REVISION="1"
+
+SOURCE_URI="https://github.com/Pulse-Eight/platform/archive/p8-platform-$portVersion.tar.gz"
+CHECKSUM_SHA256="064f8d2c358895c7e0bea9ae956f8d46f3f057772cb97f2743a11d478a0f68a0"
+SOURCE_DIR="platform-p8-platform-$portVersion"
```

Comment:
Sure it's needed?

Reply:
You mean SOURCE_DIR?

Reply:
Yes

Reply:
There would be "Command '['git', 'commit', '-m', 'import', '-q']' returned non-zero exit status 1" if having SOURCE_DIR removed.

Reply:
Strike that, needs to stay :)